src/optimizer/simple_optimizer.cpp

```diff
@@ -288,11 +297,13 @@ std::shared_ptr<planner::AbstractPlan> SimpleOptimizer::BuildPelotonPlanTree(
               ((expression::TupleValueExpression*)select_stmt->order->expr)
                   ->GetColumnName());
           for (size_t column_ctr = 0;
-               column_ctr < select_stmt->select_list->size(); column_ctr++) {
-            std::string col_name((
-                (expression::TupleValueExpression*)select_stmt->select_list->at(
-                    column_ctr))->GetColumnName());
-            if (col_name == sort_col_name) key.push_back(column_ctr);
+               column_ctr < target_table->GetSchema()->GetColumns().size(); column_ctr++) {
```

Comment:
The original code looks ugly with long lines. Can you break it up?
Like `auto schema_size = target_table->GetSchema()->GetColumns().size()` and `column_ctr < schema_size` etc.
`target_table->GetSchema()->GetColumns().at(column_ctr)`.